### DISCUSSION TITLE
The List of files is limited to 40 items

### DISCUSSION THREAD

**Discussion by RobbieRw** [mriscoc/Ender3V2S1#582] [General]
Hi Guys

I have around 150 Prints on my memory card.

However when i scroll down it goes from A down to O and then stops.

Any reason for this? Is there a limit or something? it was fine before updating my firmware.

**Comment by mriscoc** [comment]
For guarding memory use the current limit is 40 listed files

**Reply by RobbieRw** [reply]
Is there no way to remove this? Also will input shaping be automatically enabled with future release?

**Reply by mriscoc** [reply]
Only editing the configuration.h file and compiling a custom version.
https://github.com/mriscoc/Ender3V2S1/blob/72b609366965ba83826b6124878193de20c3d664/Marlin/Configuration_adv.h#L1622

Input Shaping is experimental, some binaries are available here:
https://github.com/mriscoc/Special_Configurations/releases/tag/InputShaping
